sources/platform/tutorial/runtime/ami-v594-aarch64.md

```diff
@@ -0,0 +1,98 @@
+page_main_title: AMI v5.9.4 aarch64
+main_section: Platform
+sub_section: Tutorials
+sub_sub_section: Runtime
+page_title: Description of what is supported by Machine Image v5.9.4
+page_description: A complete list of language versions, Docker versions, packages and tools available in Machine Image v5.9.4
+page_keywords: CI/CD, shippable CI/CD, documentation, shippable, config, yml, AMI, Docker
+
+# aarch64 Machine Image v5.9.4 (Docker TAG v5.9.4)
+
+**Release Date:** September 28, 2017
+
+**What is installed**
+
+* Operating System: Ubuntu 16.04
+* Kernel Version: 4.10.0-26-generic
+* **Docker Server Version: 17.06**
+* Storage Driver: aufs
+* Storage Root Dir: /var/lib/docker/aufs
+* Docker Root Dir: /var/lib/docker
+* Backing Filesystem: extfs
+* Dirperm1 Supported: true
+* Cgroup Driver: cgroupfs
+
+** Note: ONLY [custom nodes](/platform/tutorial/runtime/custom-nodes/) are currently supported. Builds on [dynamic nodes](/platform/tutorial/runtime/dynamic-nodes/) for aarch64 architecture are not supported for this version of AMI
```

Comment:
why is only in caps. did you even read my previous comment. just copy the following statemetn as-is
```
Note: Only [custom nodes](/platform/tutorial/runtime/custom-nodes/) are currently supported. Builds on [dynamic nodes](/platform/tutorial/runtime/dynamic-nodes/) for aarch64 architecture are not supported for this version of AMI **
```

Reply:
@ric03uec  sorry, updated it